Test/git-issues/git-issue-1479a.dfy

```diff
@@ -0,0 +1,14 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+type T
```

Comment:
It's fine to have the two test files, `git-issue-1479a.dfy` and `git-issue-1479b.dfy`, but if (for this PR or some future PRs) you want to combine them into one file and there are name clashes, then you can put each example in a separate `module` in the one test file.